Directory.Packages.props

```diff
@@ -8,7 +8,7 @@
     <AspireVersion>8.1.0</AspireVersion>
     <AspireUnstablePackagesVersion>8.0.0-preview.8.24258.2</AspireUnstablePackagesVersion>
     <GrpcVersion>2.64.0</GrpcVersion>
-    <DuendeVersion>7.0.5</DuendeVersion>
+    <DuendeVersion>7.0.6</DuendeVersion>
```

Comment:
^ this was package vulnerability issue; required to upgrade with `WarningsAsErrors` enabled